cmux-tui/bindings/tests/test_verify_github_environment_policy.py

```
@@ -0,0 +1,137 @@
+                    {
+                        "type": "Team",
+                        "reviewer": {"slug": "sdk-release-reviewers"},
+                    }
+                ]
+            ),
+            self.environment_name,
+            self.dispatcher,
+        )
+
+    def test_rejects_non_https_github_api_url_before_fetching(self) -> None:
+        with self.assertRaisesRegex(
```

Ruff (0.16.1)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)